src/components/User/Assets/AssetRequests/AssetRequests.Show.vue

```diff
@@ -354,10 +606,24 @@ export default {
     this.isInitializing = true
 
     try {
-      const { data } = await api.getWithSignature(`/v3/requests/${this.id}`, {
+      const { data } = await api.getWithSignature(`/v3/requests`, {
         include: ['request_details'],
       })
-      this.assetRequest = new AssetRequest(data)
+      let currentRequestIndex = data.findIndex(a => a.id === this.id)
```

Comment:
try load all asset requests(`/v3/asset_update_requests` and `/v3/create_asset_requests`) with filter by asset code, and get need  request with a computed property